### PR TITLE
mmo-finance pools yield additions

### PR DESCRIPTION
**Pull request by ohan8**
Testing logs:

`
 PASS  src/adaptors/test.js
  Running mmo-finance Test
    ✓ Check for unique pool ids (1 ms)
    ✓ Check project field is constant in all pools and if folder name and project field in pool objects matches the information in /protocols slug
    Check for allowed field names
      ✓ Expects pool id 0x9EAA40e8084F3C54B6E69C0A2ECaCF38cBf9b780 to contain only allowed keys: pool,chain,project,symbol,apy,apyBase,apyReward,underlyingTokens,rewardTokens,tvlUsd and has: pool,chain,project,symbol,tvlUsd,apy,underlyingTokens (1 ms)
      ✓ Expects pool id 0xDdf26F3529CA4dC30f484b4F9d3A7ce0b7BD1562 to contain only allowed keys: pool,chain,project,symbol,apy,apyBase,apyReward,underlyingTokens,rewardTokens,tvlUsd and has: pool,chain,project,symbol,tvlUsd,apy,underlyingTokens
      ✓ Expects pool id 0x3bca3a1e6a573ca87cddf10eb49bb508a8188cb6 to contain only allowed keys: pool,chain,project,symbol,apy,apyBase,apyReward,underlyingTokens,rewardTokens,tvlUsd and has: pool,chain,project,symbol,tvlUsd,apy,underlyingTokens
    Check apy data types
      ✓ Expects pool with id 0x9EAA40e8084F3C54B6E69C0A2ECaCF38cBf9b780 to have at least one number apy field
      ✓ Expects pool with id 0xDdf26F3529CA4dC30f484b4F9d3A7ce0b7BD1562 to have at least one number apy field
      ✓ Expects pool with id 0x3bca3a1e6a573ca87cddf10eb49bb508a8188cb6 to have at least one number apy field (1 ms)
    Check tvl data type
      ✓ tvlUsd field of pool with id 0x9EAA40e8084F3C54B6E69C0A2ECaCF38cBf9b780 should be number 
      ✓ tvlUsd field of pool with id 0xDdf26F3529CA4dC30f484b4F9d3A7ce0b7BD1562 should be number 
      ✓ tvlUsd field of pool with id 0x3bca3a1e6a573ca87cddf10eb49bb508a8188cb6 should be number  (1 ms)
    Check tokens data types
      ✓ underlyingTokens field of pool with id 0x9EAA40e8084F3C54B6E69C0A2ECaCF38cBf9b780 should be an Array of strings
      ✓ underlyingTokens field of pool with id 0xDdf26F3529CA4dC30f484b4F9d3A7ce0b7BD1562 should be an Array of strings
      ✓ underlyingTokens field of pool with id 0x3bca3a1e6a573ca87cddf10eb49bb508a8188cb6 should be an Array of strings (1 ms)
    Check other fields data types
      ✓ Expect other fields of pool with id 0x9EAA40e8084F3C54B6E69C0A2ECaCF38cBf9b780 to match thier data types
      ✓ Expect other fields of pool with id 0xDdf26F3529CA4dC30f484b4F9d3A7ce0b7BD1562 to match thier data types
      ✓ Expect other fields of pool with id 0x3bca3a1e6a573ca87cddf10eb49bb508a8188cb6 to match thier data types (1 ms)
    Check if pool id already used by other project
      ✓ Expect duplicate ids array to be empty

Test Suites: 1 passed, 1 total
Tests:       18 passed, 18 total
Snapshots:   0 total
Time:        0.232 s, estimated 1 s
Ran all test suites.

Nb of pools: 3
 

Sample pools: [
  {
    pool: '0x9EAA40e8084F3C54B6E69C0A2ECaCF38cBf9b780',
    chain: 'Ethereum',
    project: 'mmo-finance',
    symbol: 'Milady-mETH',
    tvlUsd: 1921.5728551755255,
    apy: 0.020566480505472,
    underlyingTokens: [ '0x9EAA40e8084F3C54B6E69C0A2ECaCF38cBf9b780' ]
  },
  {
    pool: '0xDdf26F3529CA4dC30f484b4F9d3A7ce0b7BD1562',
    chain: 'Ethereum',
    project: 'mmo-finance',
    symbol: 'Glasses-mETH',
    tvlUsd: 1640.9701626933709,
    apy: 0.1981995435548064,
    underlyingTokens: [ '0xDdf26F3529CA4dC30f484b4F9d3A7ce0b7BD1562' ]
  },
  {
    pool: '0x3bca3a1e6a573ca87cddf10eb49bb508a8188cb6',
    chain: 'Ethereum',
    project: 'mmo-finance',
    symbol: 'Pudgy-mETH',
    tvlUsd: 717.5551889514292,
    apy: 0.3815995770432288,
    underlyingTokens: [ '0x3bca3a1e6a573ca87cddf10eb49bb508a8188cb6' ]
  }
]
`

### CODE REVIEW

File: src/adaptors/mmo-finance/utils.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+export
```
Comment (vrtnd) [inline src/adaptors/mmo-finance/utils.js]: 🤔

**Comment by llamatester** [comment]
The mmo-finance adapter exports pools: 
        
 
 Test Suites: 1 passed, 1 total
    Tests:       18 passed, 18 total
    Snapshots:   0 total
    Time:        0.339 s
    Ran all test suites.
    
    Nb of pools: 3
     
    
    Sample pools:
    ┌─────────┬──────────────────────────────────────────────┬────────────┬───────────────┬────────────────┬────────────────────┬────────────────────┬──────────────────────────────────────────────────┐
    │ (index) │                     pool                     │   chain    │    project    │     symbol     │       tvlUsd       │        apy         │                 underlyingTokens                 │
    ├─────────┼──────────────────────────────────────────────┼────────────┼───────────────┼────────────────┼────────────────────┼────────────────────┼──────────────────────────────────────────────────┤
    │    0    │ '0x9EAA40e8084F3C54B6E69C0A2ECaCF38cBf9b780' │ 'Ethereum' │ 'mmo-finance' │ 'Milady-mETH'  │ 2011.2263133306135 │ 0.020566480505472  │ [ '0x9EAA40e8084F3C54B6E69C0A2ECaCF38cBf9b780' ] │
    │    1    │ '0xDdf26F3529CA4dC30f484b4F9d3A7ce0b7BD1562' │ 'Ethereum' │ 'mmo-finance' │ 'Glasses-mETH' │ 1717.5317405791802 │ 0.1981995435548064 │ [ '0xDdf26F3529CA4dC30f484b4F9d3A7ce0b7BD1562' ] │
    │    2    │ '0x3bca3a1e6a573ca87cddf10eb49bb508a8188cb6' │ 'Ethereum' │ 'mmo-finance' │  'Pudgy-mETH'  │ 751.0336511046359  │ 0.3815995770432288 │ [ '0x3bca3a1e6a573ca87cddf10eb49bb508a8188cb6' ] │
    └─────────┴──────────────────────────────────────────────┴────────────┴───────────────┴────────────────┴────────────────────┴────────────────────┴──────────────────────────────────────────────────┘

**Comment by 0xvrotend** [comment]
Hello, thank you for your PR.
1) Please convert apy value to %
2) If the pool reward is a token then change `apy` to `apyReward` and add `rewardTokens` field

**Comment by ohan8** [comment]
Just pushed new changes with APY as percentage and removed excess utils file. The APY is from supplying and not as LM rewards so thats why I kept it as APY base and not rewards. Let me know if this is fine. Thanks!

**Comment by llamatester** [comment]
The mmo-finance adapter exports pools: 
        
 
 Test Suites: 1 passed, 1 total
    Tests:       18 passed, 18 total
    Snapshots:   0 total
    Time:        0.383 s
    Ran all test suites.
    
    Nb of pools: 3
     
    
    Sample pools:
    ┌─────────┬──────────────────────────────────────────────┬────────────┬───────────────┬────────────────┬────────────────────┬───────────────────┬──────────────────────────────────────────────────┐
    │ (index) │                     pool                     │   chain    │    project    │     symbol     │       tvlUsd       │        apy        │                 underlyingTokens                 │
    ├─────────┼──────────────────────────────────────────────┼────────────┼───────────────┼────────────────┼────────────────────┼───────────────────┼──────────────────────────────────────────────────┤
    │    0    │ '0x9EAA40e8084F3C54B6E69C0A2ECaCF38cBf9b780' │ 'Ethereum' │ 'mmo-finance' │ 'Milady-mETH'  │ 2067.3607401103495 │  2.0566480505472  │ [ '0x9EAA40e8084F3C54B6E69C0A2ECaCF38cBf9b780' ] │
    │    1    │ '0xDdf26F3529CA4dC30f484b4F9d3A7ce0b7BD1562' │ 'Ethereum' │ 'mmo-finance' │ 'Glasses-mETH' │ 1765.4689911483388 │ 19.81995435548064 │ [ '0xDdf26F3529CA4dC30f484b4F9d3A7ce0b7BD1562' ] │
    │    2    │ '0x3bca3a1e6a573ca87cddf10eb49bb508a8188cb6' │ 'Ethereum' │ 'mmo-finance' │  'Pudgy-mETH'  │  771.995411209711  │ 38.15995770432288 │ [ '0x3bca3a1e6a573ca87cddf10eb49bb508a8188cb6' ] │
    └─────────┴──────────────────────────────────────────────┴────────────┴───────────────┴────────────────┴────────────────────┴───────────────────┴──────────────────────────────────────────────────┘

**Comment by 0xvrotend** [comment]
Merged, but just fyi pools will be visible on UI only if tvl >10k$

**Pull request merged**